src/http/conn.rs

```diff
@@ -490,7 +497,7 @@ enum Reading {
 
 enum Writing<B> {
     Init,
-    Body(Encoder, Option<Cursor<B>>),
+    Body(Encoder, Option<Cursor<B>>, BytesMut),
```

Comment:
Yeah fair enough. I was hoping that if we could move all this to the Tokio types then the Tokio foundation would magically handle all the performance stuff.

Whats a good way to run performance tests? I only really have access to a 4 year old mackbook pro